bftengine/src/bftengine/messages/ViewChangeMsg.cpp

```diff
@@ -178,6 +216,32 @@ bool ViewChangeMsg::checkElements(uint16_t sigSize) const {
   return true;
 }
 
+bool ViewChangeMsg::checkComplaints(uint16_t sigSize) const {
```

Comment:
This method looks more like `checkComplaintsListIntegrity()`. I don't see any signature verifications here?

Reply:
Yes, here we only check the integrity of the list of complaints. The actual check will be carried at a higher level.